### PR TITLE
Adds hereprisoner

### PR DESCRIPTION
**Pull request by BoundCNapalm**
## About The Pull Request

Allows heretic prisoner by removing JOB_PRISONER from protected roles.

## Why It's Good For The Game

Prisoners are already being monitored to begin with, so you can pretty easily get rid of heretic then. Heretic prisoner has cool roleplay opportunities. Especially if a chaplain comes in to exorcise the prisoner. 

## Changelog

All this does is allow heretic prisoner spawns by removing them from the protected roles list. 

:cl:
code: Removes two of the same entries in the heretic spawns, JOB_PRISONER.
config: Adds natural heretic prisoner spawns.
/:cl:

### CODE REVIEW

**Comment by nevimer** [comment]
nonmodular undocumented remove, please comment it out instead

**Comment by BoundCNapalm** [comment]
Will add commits soon to revert it

File: code/game/gamemodes/dynamic/dynamic_rulesets_roundstart.dm [hunk 1]
```diff
@@ -212,7 +212,7 @@ GLOBAL_VAR_INIT(revolutionary_win, FALSE)
 		JOB_CAPTAIN,
 		JOB_DETECTIVE,
 		JOB_HEAD_OF_SECURITY,
-		JOB_PRISONER,
+		//JOB_PRISONER,
```
Comment (BurgerLUA) [inline code/game/gamemodes/dynamic/dynamic_rulesets_roundstart.dm:215]: This needs to be indicated that it is a modular change.

**Review by BurgerLUA — CHANGES_REQUESTED** [review]

File: code/game/gamemodes/dynamic/dynamic_rulesets_roundstart.dm (hunk 1 above)
Comment (StrangeWeirdKitten) [inline code/game/gamemodes/dynamic/dynamic_rulesets_roundstart.dm:215]:
```suggestion
		//JOB_PRISONER, - BUBBERSTATION REMOVAL
```

File: code/game/gamemodes/dynamic/dynamic_rulesets_latejoin.dm [hunk 2]
```diff
@@ -206,7 +206,7 @@
 		JOB_DETECTIVE,
 		JOB_HEAD_OF_PERSONNEL,
 		JOB_HEAD_OF_SECURITY,
-		JOB_PRISONER,
+		//JOB_PRISONER, ## BUBBER EDIT REMOVAL
```
Comment (StrangeWeirdKitten) [inline code/game/gamemodes/dynamic/dynamic_rulesets_latejoin.dm:209]:
```suggestion
		//JOB_PRISONER, - BUBBERSTATION REMOVAL
```

**Comment by StrangeWeirdKitten** [comment]
Basic edits aside, this can be test merged.

**Comment by projectkepler-ru** [comment]
Stale

**Pull request closed without merging**